docs/blog/posts/distilation-part1.md

```diff
@@ -103,26 +103,45 @@ Here's how the logging output would look:
 
 Run a finetune like this:
 
+!!! note annotate "Don't forget to set your OpenAI Key as an environment variable"
+
+    All of the `instructor jobs` commands assume you've set an environment variable of `OPENAI_API_KEY` in your shell. You can set this by running the command `export OPENAI_API_KEY=<Insert API Key Here>` in your shell
+
 ```bash
 instructor jobs create-from-file math_finetunes.jsonl
 ```
 
 ## Next Steps and Future Plans
 Here's a sneak peek of what I'm planning:
 
+
 ```python
-from instructor import Instructions
+from instructor import Instructions,patch
```

Comment:
@ivanleomk in general you should try to install ruff or some formatter in your ide

```suggestion
from instructor import Instructions, patch
```